double_pendulum:
  ros__parameters:
    publish_freq: 10.0
    l1: 0.08
    l2: 0.2
    m1: 5.0
    m2: 3.0
    a1_0: 30.0
    a2_0: 10.0
    v1_0: 0.0
    v2_0: 0.0
    T: 10.0
    h: 0.01